indigo/docs/guides/howto-indigo-game.md

```diff
@@ -0,0 +1,194 @@
+---
+id: howto-indigo-game
+title: Converting `IndigoSandbox` to `IndigoGame`
+---
+
+## Basic Concepts
+
+You may have noticed that many of our tutorials and examples make use of
+`IndigoSandbox` and `IndigoDemo`. But what if you want to go further than these
+examples? How would you add scene management, or deal with sub systems? In these
+cases it's time to move your game to use `IndigoGame`, which comes with a host
+of features, but also a lot more boiler-plate code.
+
+Note: In this tutorial we'll be moving our code to `IndigoGame` and using an
+empty scene, but we won't be making full use of the scene management.
+If you want to know more about managing scenes in Indigo you can read the
+documentation [here](https://indigoengine.io/docs/organisation/scene-management).
```

Comment:
This is fine, but just for future reference there is an internal link format, you just do `[here](scene-management.md)` (I think), and docusaurus does the rest.